services/repository/src/main/java/org/sagebionetworks/repo/web/UrlHelpers.java

```diff
@@ -921,6 +920,7 @@
 	public static final String NAME_FRAGMENT_FILTER = "fragment";
 	public static final String MEMBER_TYPE_FILTER = "memberType";
 	public static final String TEAM_ID_ICON = TEAM_ID+"/icon";
+	public static final String TEAM_ID_ICON_PREVIEW = TEAM_ID+"/icon/preview";
```

Comment:
We can reuse the TEAM_ID_ICON constant: TEAM_ID_ICON_PREVIEW  = TEAM_ID_ICON  + "/preview";